flink-runtime-web/web-dashboard/.eslintrc.js

```diff
@@ -126,7 +126,9 @@ module.exports = {
     {
       files: ['*.html'],
       extends: ['plugin:@angular-eslint/template/recommended'],
-      rules: {}
+      rules: {
+        '@angular-eslint/template/eqeqeq': 'warn'
```

Comment:
Don't do this. You shouldn't allow Abstract Equality Comparison, and always prefer Strict Equality Comparison for being type-safe.